### PR TITLE
Fix to ignore SCSS namespace

### PR DESCRIPTION
**Pull request by kimulaco**
Issue: Rules are not ignored when using SCSS namespace (https://github.com/stylelint/stylelint/issues/4378)

There is a rule to ignore SCSS variable in "lib/utils/isStandardSyntaxValue.js".
I added a rule there that also ignores the SCSS namespace.

### CODE REVIEW

**Comment by kimulaco** [comment]
I added the following test.

- Accept LESS variable
- Accept SCSS variable
- Accept SCSS namespace

Verification, please.

**Comment by hudochenkov** [comment]
Please, add tests to [isStandardSyntaxValue.test.js](https://github.com/stylelint/stylelint/blob/master/lib/utils/__tests__/isStandardSyntaxValue.test.js) as well. And merge `master` into your branch. We fixed issue with our CI in master and would like to see that CI is passes for your PR.

File: lib/utils/isStandardSyntaxValue.js [hunk 1]
```diff
@@ -14,11 +14,16 @@ module.exports = function(value /*: string*/) /*: boolean*/ {
...
 	}
 
-	// SCSS variable
+	// SCSS variable (example $variable)
 	if (normalizedValue[0] === '$') {
 		return false;
 	}
 
+	// SCSS namespace (example namespace.$variable)
+	if (/^.*\.\$/.test(value)) {
```
Comment (hudochenkov) [inline lib/utils/isStandardSyntaxValue.js]:
```suggestion
	if (/^.+\.\$/.test(value)) {
```

Because namespace should be at least one character long.

Comment (kimulaco) [reply lib/utils/isStandardSyntaxValue.js]: Thanks! I fixed it.

**Comment by kimulaco** [comment]
Added a test to "isStandardSyntaxValue.test.js".
And merged from a master, CI was also successful!

**Comment by hudochenkov** [comment]
> -   Fixed: `font-family-no-missing-generic-family-keyword` false positives for Sass-variables with namespaces ([#4378](https://github.com/stylelint/stylelint/pull/4378)).

**Pull request merged**